### ISSUE TITLE
[Feature Request]: Dedicated setting for claiming GUI

### ISSUE THREAD

**Issue by Proxwian** [FTBTeam/FTB-Mods-Issues#797]
### Mod

FTB Chunks

### Description

Currently, FTBChunks does not allow to partially limit player access to the large map, and have same options for displaying both GUIs: fullscreen map (key M by default) and claiming gui (key C while opened fullscreen map).

### Describe the solution you'd like

It would be great if we can disable fullscreen map via separate gamestage or via server config. And also have separate button (same as logic) for claiming GUI. It would add more compatibility for mods like AntiqueAtlas and MapAtlas.

### Describe alternatives you've considered

Currently there is only one option: to disable map entirely, by enabling gamestage limitation via config.

**Comment by desht** [comment]
Yeah, that would make sense. There should be better independence between the claim screen and the large map screen (while preserving existing functionality of switching between them).

**Comment by Proxwian** [comment]
@desht as I can see, there is no separate gamestage option for having only claiming GUI enabled (and having big map with poi disabled). Is there any plans for it?

**Comment by desht** [comment]
Does there need to be a separate gamestage for this?  As it is now, players can still use the chunk claiming GUI even if they don't have the gamestage for the big map.  That's what you wanted, isn't it?

**Comment by Proxwian** [comment]
Maybe I look it wrong.. BTW, is it coming for 1.18.2 too?

**Comment by Elnhomantia** [comment]
Hi, I'm stumbling on this issue while I'm trying to achieve the original author's goal, but can't seems to be able to do it.
By adding 'ftbchunks_mapping' stage to a player, he's able to use both the big and the claim map, while I would only like the claim map active as I plan to use map atlases. @desht seems to suggest that by adding the 'ftbchunks_mapping' stage to a player, only the claim map should be accessible, which is not the resul I observe. Did I miss something ?

(I forgot to give my version)
Neofoge 1.21.1
ftbchunk 2101.1.15

**Comment by desht** [comment]
I think there may have been a regression here - the chunk claim screen is indeed currently unavailable without the game stage.  Looks like a bug - it should be available.

**Comment by SonofRanman66** [comment]
@desht Tested in 1.21.1 NeoForge using the recent FTB Chunk update, you can still access the Large Map from the Claim Manager GUI if you click the Large Map Button, even when the Require Mapping Game state is set to True

<img width="1437" height="1060" alt="Image" src="https://github.com/user-attachments/assets/b372fa50-9d30-4ad2-9403-7bebc2161aa9" />

**Comment by SonofRanman66** [comment]
@desht Just realized, it might be worth adding the Server Settings, and regular Settings buttons to the Claimed Chunks GUI, as it's only accessible on the Large Map GUI, and is needed to manage default claim settings.

<img width="404" height="164" alt="Image" src="https://github.com/user-attachments/assets/de5e55a2-0dac-48bd-85f6-d84e99c432d8" />

<img width="239" height="126" alt="Image" src="https://github.com/user-attachments/assets/5aafa431-5e76-4ddc-87f2-14286e33dd09" />

**Comment by desht** [comment]
Fixed in 2101.1.19 release
